### DISCUSSION TITLE
Sera vue o react ?👀

### DISCUSSION THREAD

**Discussion by snowingt** [Launch-X-Latam/MisionFrontEnd#81] [Q&A]
@romarpla  al final que aprenderemos vue o react ?

**Comment by Emilio17MC** [comment]
El Mission Commander comentó durante el live de bienvenida que cuando llegue el momento se hará una encuesta para decidir cuál veremos.

**Reply by snowingt** [reply]
Vale no se como me salte esa parte volvere a ver el live , gracias de todos modos 💖

**Comment by EsthebanDiego** [comment]
Hicieron encuestas aqui en Git y parece que la tendencia va mas para React

**Reply by FaraN1ght** [reply]
Ojalá Dios y el Lord Comandante te escuchen, ando estudiando React por Udemy y me vendría bien el bootcamp con React.

**Reply by snowingt** [reply]
Yo estoy aprendiendo react  full empece igual me conviene antes de ya habia probado un poco vue para ver diferencia pero react me parece bien
